packages/server/utils/analytics/helpers.ts

```diff
@@ -1,10 +1,25 @@
+import {CHECKIN} from '../../../client/utils/constants'
+import Meeting from '../../database/types/Meeting'
+import MeetingMember from '../../database/types/MeetingMember'
 import MeetingTemplate from '../../database/types/MeetingTemplate'
 
-export const createMeetingTemplateAnalyticsParams = (template: MeetingTemplate) => {
+export const createMeetingProperties = (
+  completedMeeting: Meeting,
```

Comment:
+1 This isn't necessarily a completed meeting right?  E.g. in the case of "meeting joined"? We should probably change this to just `meeting`

Reply:
Good point! Will rename